bundles/org.openhab.ui/doc/components/oh-map-page.md

```diff
@@ -52,12 +52,12 @@ Displays markers on a map
 </PropBlock>
 <PropBlock type="TEXT" name="tileLayerProvider" label="Provider for the background tiles">
   <PropDescription>
-    The provider of tiles to use for the background of the map. Use one from <a class="external text-color-blue" target="_blank" href="https://leaflet-extras.github.io/leaflet-providers/preview/">Leaflet Providers</a>, Some providers will not work until you set options, like access tokens, in the <code>tileLayerProviderOptions</code> parameter (in Code view). See <a class="external text-color-blue" target="_blank" href="https://github.com/leaflet-extras/leaflet-providers#providers-requiring-registration">here</a> for more info. The default is CartoDB, the variant depending on the dark mode setting.
+    The provider of tiles to use for the background of the map. Use one from <a class="external text-color-blue" target="_blank" href="http://leaflet-extras.github.io/leaflet-providers/preview/">Leaflet Providers</a>, Some providers will not work until you set options, like access tokens, in the <code>tileLayerProviderOptions</code> parameter (in Code view). See <a class="external text-color-blue" target="_blank" href="https://github.com/leaflet-extras/leaflet-providers#providers-requiring-registration">here</a> for more info. The default is CartoDB, the variant depending on the dark mode setting.
```

Comment:
looks like you partly reverted https://github.com/openhab/openhab-webui/pull/1589 here!

Reply:
Never mind, actually it was #1589 that wasn't done right ;)